docs/spec_draft.md

```diff
@@ -0,0 +1,124 @@
+# StableHLO Specification Draft
+
+## Types
+
+Following are the supported element types in StableHLO:
+
+  * **Integer types**
+
+    * Signed integer with two’s complement representation. Referred to in the
+    document as `si<N>`, where the bit-width N ∊ {4, 8, 16, 32, 64}
+    * Unsigned integer referred to in the document as `ui<N>`, where the
+    bit-width N ∊ {4, 8, 16, 32, 64}
+  * **Boolean types** referred to in the document as `pred`
+  * **Floating point types**
```

Comment:
I think the canonical spelling is "Floating-point types".

Reply:
updated.